shared/middleware/helpers/animation_bucket.rb

```diff
@@ -24,17 +24,17 @@ def cache_duration_seconds
   # and we want to see these incidents go down over time.
   #
   def s3_get_object(key, if_modified_since, version)
-    super(key, if_modified_since, version)
+    requested_version = version
+    if version === 'latestVersion'
```

Comment:
Triple-equals [does not do what you think](https://stackoverflow.com/questions/4467538/what-does-the-operator-do-in-ruby) in Ruby.  `==` is sufficient here. :joy:

Reply:
Javascript habits are strong. :)